src/Parser/DeathKnight/Blood/Modules/Features/Checklist.js

```diff
@@ -140,16 +138,12 @@ class Checklist extends CoreChecklist {
           new Requirement({
             name: <React.Fragment><SpellLink id={SPELLS.MARK_OF_BLOOD_TALENT.id}/> Uptime</React.Fragment>,
             check: () => this.markOfBloodUptime.uptimeSuggestionThresholds,
+            when: this.combatants.selected.hasTalent(SPELLS.MARK_OF_BLOOD_TALENT.id),
```

Comment:
Shouldn't this change be made to live instead?

Reply:
I clearly read the PRs in wrong order. Should be able to leave this out however, since zero periodically merges master into BFA.